### ISSUE TITLE
How to integrate the phantasmachain in own projects?

### ISSUE THREAD

**Issue by furesoft** [phantasma-io/PhantasmaChain#58]
How can I use this chain to make custom p2p applications?

**Comment by Relfos** [comment]
Hi @furesoft , what kind of p2p application you want to make?

**Comment by furesoft** [comment]
I want to integrate the phantasma chain as internal currency for my integrated digital asset store. I also want this chain to store where the assets are stored or store the assets direktly on to the chain.

**Comment by Relfos** [comment]
Yes, this seems like a good use for Phantasma chain, check the docs at:
https://docs.phantasma.io

For more direct and faster communication you can also join our Discord server:
https://discord.gg/sUjcHFk68t

**Comment by GameFemx** [comment]
> How can I use this chain to make custom p2p applications?

It easy, you can solve this issue via live chat https://direct.lc.chat/14345826/
